packages/component-library/stories/IconMap.story.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/react';
+import { withKnobs, number, selectV2, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { checkA11y } from '@storybook/addon-a11y';
+import { BaseMap } from '../src';
+import { IconMap } from '../src';
+
+import data from '../src/IconMap/data.json';
+
+const displayName = IconMap.displayName || 'IconMap';
+
+const mapboxToken = 'pk.eyJ1IjoidGhlbWVuZG96YWxpbmUiLCJhIjoiY2o1aXdoem1vMWtpNDJ3bnpqaGF1bnlhNSJ9.sjTrNKLW9daDBIGvP3_W0w';
+
+const demoMap = () => {
+  const optionsStyle = {
+    'Lè Shine': 'mapbox://styles/themendozaline/cjg6296ub04ot2sqv9izku3qq',
+    'Label Maker': 'mapbox://styles/themendozaline/cjg627xuw08mk2spjsb8jmho7',
+    'Moonlight': 'mapbox://styles/themendozaline/cjgq6r2lg00072rmqj1wocgdq',
+    'Navigation Guidance Night': 'mapbox://styles/themendozaline/cj6y6f5m006ar2sobpimm7ay7',
+    'North Star': 'mapbox://styles/themendozaline/cj5oyewyy0fg22spetiv0hap0',
+    'Odyssey': 'mapbox://styles/themendozaline/cjgq6rklb000d2so1b8myaait',
+    'Scenic': 'mapbox://styles/themendozaline/cj8rrlv4tbtgs2rqnyhckuqva',
+  };
+  const mapboxStyle = selectV2('Mapbox Style', optionsStyle, optionsStyle['Scenic']);
+
+  const opacityOptions = {
+     range: true,
+     min: 0,
+     max: 1,
+     step: 0.1,
+  };
+  const opacity = number('Opacity:', 1, opacityOptions);
+
+  const iconAtlas = 'https://i.imgur.com/29Kt7Ii.png';
+
+  const iconMapping = {
+    "School": {
+      "x": 0,
+      "y": 0,
+      "width": 250,
+      "height": 250,
+      "mask": true,
+    },
+    "Hospital": {
+      "x": 250,
+      "y": 0,
+      "width": 250,
+      "height": 250,
+      "mask": true,
+    },
+    "Fire Station": {
+      "x": 0,
+      "y": 250,
+      "width": 250,
+      "height": 250,
+      "mask": true,
+    },
+    "Pin": {
+      "x": 250,
+      "y": 250,
+      "width": 250,
+      "height": 250,
+      "mask": true,
+    },
+  };
+
+  const iconSizeScaleOptions = {
+     range: true,
+     min: 1,
```

Comment:
Consider `min: .5`, or maybe even smaller. It's kind of nice in the storybook to see what is possible, and it's possible to make these icons smaller. Definitely a nice-to-have, not essential.

Reply:
:+1: I'll definitely make that change in the next iteration